### PR TITLE
Adding RxDb logo

### PR DESCRIPTION
**Pull request by pete-hotchkiss**
RxDB

**Issue:** #5620 
**Alexa rank:** n/a
**GitHub stars:** 15.5k

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Brand has 3 main colours is logo - so picked the hex applied to the accompanying text

### CODE REVIEW

**Comment by pete-hotchkiss** [comment]
@adamrusted anything else you need from me to wrap up this PR ?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7551,6 +7551,11 @@
                 "type": "CC-BY-SA-4.0"
             }
         },
+        {
+            "title": "RxDb",
```
Comment (adamrusted) [inline _data/simple-icons.json:7555]: Though it doesn't really matter in the grand scheme of things, the brand is stylized as 'RxDB'.
```suggestion
            "title": "RxDB",
```

File: icons/rxdb.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"><title>RxDb icon</title><path d="M4.2 15.6v4.9h6.4v1.2h1.2v-1.2h1.1v2.3h1.2v-2.3h1.1v2.3h1.2V24h1.1v-2.3h1.2v1.1h1.1v-7.2z M4.2 14.5h15.6v-5H4.2v5z M6.5 0v2.3H5.3V1.2H4.2v7.2h15.6V3.5h-6.4V2.3h-1.2v1.2h-1.1V1.2H9.9v2.3H8.8V1.2H7.6V0z"/></svg>
```
Comment (adamrusted) [inline icons/rxdb.svg:1]: I'm seeing a [few differences](https://user-images.githubusercontent.com/1830867/117814264-fbba7100-b25b-11eb-98ad-c22dc83f9490.png) between your version in red, and the source icon in black. Would you mind taking another look at this, and don't forget to update the title to 'RxDB' to match the JSON!

Comment (pete-hotchkiss) [reply icons/rxdb.svg]: OK - thats updated. I'm not sure what tool you we using to do the diffs between the two versions but the `.svg` file does have updated path nodes

**Review by adamrusted — COMMENTED** [review]
Wow, okay. I didn't realise just how messy the source SVG was (collinear points, etc). I've been able to get a tidy 3-decimal path working on the linter, which you're welcome to use as there were still some differences between yours and source. If you do use this path though, we'll need to request another maintainer reviews this.

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>RxDB icon</title><path d="M11.096 3.452h1.15v-1.15h1.151v1.15h6.411v4.932H4.192V1.15h1.15V2.3h1.151V0h1.15v1.15h1.152v2.302h1.15V1.151h1.151v2.301zM4.192 14.466h15.616V9.534H4.192v4.932zm15.616 1.15H4.192v4.932h6.41v1.15h1.151v-1.15h1.151v2.301h1.15v-2.301h1.152v2.301h1.15V24h1.151v-2.301h1.15v1.15h1.151v-7.233z"/></svg>
```

**Review by PeterShaggyNoble — APPROVED** [review]
Everything looks good to me, @pete-hotchkiss :+1: Thanks for the contribution and thanks @adamrusted for the help.

**Pull request merged**